build.gradle

```diff
@@ -32,32 +33,38 @@ buildscript {
 
 subprojects {
   afterEvaluate { project ->
-    task sourcesJar(type: Jar) {
-      from android.sourceSets.main.java.srcDirs
-      classifier = 'sources'
-    }
 
-    task javadoc(type: Javadoc) {
-      source = android.sourceSets.main.java.srcDirs
-      classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
-      classpath += fileTree(dir: "$buildDir/intermediates/exploded-aar/", include: "**/classes.jar")
-      classpath += configurations.compile
-      // Exclude generated files
-      exclude '**/BuildConfig.java'
-      exclude '**/R.java'
-    }
+    // These two projects are JARed and DOCed using their own build.gradle files
+    if (!project.projectDir.absolutePath.contains("backbone") && !project.projectDir.absolutePath.contains("skin")) {
```

Comment:
I like the idea of submodules, but needing to do things like this makes me wonder if we're not using gradle correctly. I briefly looked into this in December but couldn't find examples of how to do nested builds that should also work as standalone builds. It imagine it's possible because what we're trying to do with these submodules is reasonable and pretty common

I think the reason this happens is we're essentially ignoring ResearchStack's top level build.gradle (which is bad for many reasons). And likewise, mPower is ignore this build.gradle and also changing the pathing, which is why we had the issue of  not being able to find artifact-deploy script. When we enter a submodule, it'd expect gradle to pick up that submodule's top level build script and use it to build that submodule